packages/block-library/src/navigation/edit/placeholder/index.js

```diff
@@ -39,79 +46,91 @@ export default function NavigationPlaceholder( {
 		onFinish( navigationMenu, blocks );
 	};
 
-	const {
-		isResolvingPages,
-		isResolvingMenus,
-		hasMenus,
-	} = useNavigationEntities();
-
-	const isStillLoading = isResolvingPages || isResolvingMenus;
+	const { hasMenus, isResolvingMenus } = useNavigationEntities();
 
 	const onCreateEmptyMenu = () => {
 		onFinishMenuCreation( [] );
 	};
 
 	const { navigationMenus } = useNavigationMenu();
 
+	useEffect( () => {
+		if ( isResolvingMenus ) {
+			speak(
```

Comment:
@getdave Does this only fire on selectBlock()? If not, users with more than 1 Navigation Block could be in for a not so great experience with multiple messages being read.

Reply:
Great catch. I've added this. One to remember for the future. Thank you 🙇‍♂️